skbio/diversity/alpha/_base.py

```diff
@@ -847,6 +848,51 @@ def osd(counts):
     counts = _validate_counts_vector(counts)
     return observed_otus(counts), singles(counts), doubles(counts)
 
+@experimental(as_of="0.4.0")
```

Comment:
Thanks for the input @gregcaporaso, @ebolyen!

I did run `make test` and assumed it was fine when I received no errors from the code. I did receive an error with flake8, like you mentioned.
```
Ran 1313 tests in 66.137s

OK
flake8 skbio setup.py checklist.py
make: flake8: No such file or directory
make: *** [test] Error 1
```

How can I fix this?